packages/react-app-rewire-eslint/index.js

```diff
@@ -1,7 +1,27 @@
+const path = require('path');
+const { getLoader } = require('react-app-rewired');
+
+// this is the path of eslint-loader `index.js`
+const ESLINT_PATH = `eslint-loader${path.sep}index.js`;
```

Comment:
Why not to use require.resolve?

Reply:
use the same style with other packages. 
see: https://github.com/timarney/react-app-rewired/blob/master/packages/react-app-rewire-less/index.js#L8

Reply:
ok let's stick to it for now